libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -64,19 +65,51 @@ public CriStoringService(
         VcHelper.setConfigService(configService);
     }
 
-    public void storeCriResponse(
+    public void recordCriResponse(
+            JourneyRequest journeyRequest,
+            String criId,
+            String criOAuthSessionId,
+            ClientOAuthSessionItem clientOAuthSessionItem)
+            throws SqsException, JsonProcessingException {
+
+        recordCriResponse(
+                criId,
+                criOAuthSessionId,
+                journeyRequest.getIpvSessionId(),
+                journeyRequest.getIpAddress(),
+                journeyRequest.getDeviceInformation(),
+                clientOAuthSessionItem);
+    }
+
+    public void recordCriResponse(
```

Comment:
not a huge fan of overloading, is there a reason to not just update the callers to align the method calls instead?

Reply:
As discussed we're OK to leave this as it would be messy to remove the overload